plugins/kotlin/j2k/k1.new/src/org/jetbrains/kotlin/nj2k/printing/JKCodeBuilder.kt

```diff
@@ -574,7 +576,8 @@ internal class JKCodeBuilder(context: NewJ2kConverterContext) {
             ktConvertedFromForLoopSyntheticWhileStatement: JKKtConvertedFromForLoopSyntheticWhileStatement
         ) {
             printer.renderList(
-                ktConvertedFromForLoopSyntheticWhileStatement.variableDeclarations, ::ensureLineBreak) {
+                ktConvertedFromForLoopSyntheticWhileStatement.variableDeclarations, ::ensureLineBreak
```

Comment:
This is a pure formatting change, please put such changes into a separate commit. Also, the correct formatting here is:

```
 printer.renderList(ktConvertedFromForLoopSyntheticWhileStatement.variableDeclarations, ::ensureLineBreak) {
    it.accept(this)
}
```